pom.xml

```diff
@@ -185,7 +185,15 @@
               <outputDirectory>${basedir}/target/site/examples</outputDirectory>
               <resources>
                 <resource>
-                  <directory>src/examples</directory>
+                  <directory>src/test/java/org/apache/hc/core5/http/examples</directory>
```

Comment:
@garydgregory May I suggest that `maven-resources-plugin` section be moved to individual module poms? Module specific stuff in the main project pom looks a bit dodgy now.

Reply:
@ok2c ,

Thank you for the review. I committed your requested changes.

Gary

Reply:
I must have messed up the squashing. I think it is easier to squash in GH when you merge. Can we do it that way?